partner_communication/models/communication_job.py

```diff
@@ -244,7 +244,7 @@ def _inverse_ir_attachment_tmp(self):
                     }
                 )
             job.ir_attachment_ids = [
-                (4, attachment.id) for attachment in job.ir_attachment_tmp
+                (6, attachment.id) for attachment in job.ir_attachment_tmp
```

Comment:
This cannot be correct, I think the old version was correct.

Reply:
yes you are right i put it back as before.